pipelines/deploy-fbc-operator/0.1/README.MD

```diff
@@ -157,6 +161,10 @@ Name of the Kubernetes Secret that contains the access token for a private GitHu
 
 Key within the Secret's `data` field that holds the GitHub/GitLab access token.
 
+#### PATH_TO_MIRROR_SET (Optional)
+
+Path to where the image digest mirror set is saved in the repository.
+
```

Comment:
Adding the PATH_TO_MIRROR_SET section at this line breaks the previous section (REPO_KEY). Could you please move the PATH_TO_MIRROR_SET section just after line 175, before the new section ('What Happens Next') begins?

Reply:
of course. I completely missed the example being associated with the REPO_KEY. Sorry about that and thanks for spotting it.